# Motor status message from the motor sensors in Albatros motor board.
# Status for each motor is the cumulative count of failures.

Header header
int32[4] status

